From 9de86f91b9971108a2d7e1b0901c1d3bf1b974f9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <pbrady@redhat.com>
Date: Tue, 2 Apr 2013 14:34:54 +0100
Subject: [PATCH] remove deprecated assert_unicode sqlalchemy attribute

Removing this enables use with sqlalchemy 0.8.0
The deprecation is described upstream at:
http://docs.sqlalchemy.org/en/latest/changelog/changelog_08.html#change-ad44af79c886d1bb283042deb64f9cbe

Change-Id: Ia34b38245c6abe1ddb4162eee89459a7130deacb
---
 glance/db/sqlalchemy/migrate_repo/schema.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/glance/db/sqlalchemy/migrate_repo/schema.py b/glance/db/sqlalchemy/migrate_repo/schema.py
index 1fcf845..54f8808 100644
--- a/glance/db/sqlalchemy/migrate_repo/schema.py
+++ b/glance/db/sqlalchemy/migrate_repo/schema.py
@@ -28,12 +28,12 @@ LOG = logging.getLogger(__name__)
 
 
 String = lambda length: sqlalchemy.types.String(
-    length=length, convert_unicode=False, assert_unicode=None,
+    length=length, convert_unicode=False,
     unicode_error=None, _warn_on_bytestring=False)
 
 
 Text = lambda: sqlalchemy.types.Text(
-    length=None, convert_unicode=False, assert_unicode=None,
+    length=None, convert_unicode=False,
     unicode_error=None, _warn_on_bytestring=False)
 
 
